### ISSUE TITLE
pull in examples from p5js.org

### ISSUE THREAD

**Issue by therewasaguy** [therewasaguy/p5js-webIDE#4]
The desktop editor pulls in examples from p5js.org, and we should mirror this task https://github.com/processing/p5.js-editor/blob/master/gulpfile.js#L172

**Comment by therewasaguy** [comment]
examples are there, but still to do: 
- [ ] fetch examples from p5js.org repo with a gulp task / as part of the build script, instead of including them in this repo, to stay in sync
- [ ] some examples require external assets...include them somehow (this is part of a bigger issue of how to work with external assets)
- [ ] the loadExample method required a hack to reset the ace editor content, which should not have been necessary. Figure out why.

**Comment by shiffman** [comment]
Would it make sense to have examples as entries in the database, perhaps with a "p5js" user account?  I'm not advocating for this, just wondering out loud.  Would be happy to discuss the external assets question, I've been running into this issue with embedding examples into learningprocessing.com site.

**Comment by therewasaguy** [comment]
Yes, I like the idea of a p5 user, maybe a learningprocessing user as well. This could also help keep track of forks in the long run. What I did today was more of a temporary solution just to get examples in there at all—they really add a lot
